x-pack/plugin/security/src/test/java/org/elasticsearch/test/SecuritySettingsSource.java

```diff
@@ -60,8 +60,19 @@
     public static final String TEST_USER_NAME = "test_user";
     public static final Hasher HASHER = getFastStoredHashAlgoForTests();
     public static final String TEST_PASSWORD_HASHED = new String(HASHER.hash(new SecureString(TEST_PASSWORD.toCharArray())));
+
     public static final String TEST_ROLE = "user";
-    public static final String TEST_SUPERUSER = "test_superuser";
+    private static final String TEST_ROLE_YML = """
+        user:
+          cluster: [ ALL ]
+          indices:
+            - names: '*'
+              allow_restricted_indices: true
+              privileges: [ ALL ]
+        """;
+
+    public static final String TEST_UNRESTRICTED_USER = "test_root_user";
```

Comment:
Similar to above, is it better if we consolidate to always use something like `es_test_root_user` for both variable name and value?